pkg/net/net.go

```diff
@@ -66,3 +70,24 @@ func NthIPInNetwork(network *net.IPNet, n int) (net.IP, error) {
 
 	return nil, errors.New("network does not contain enough IPs")
 }
+
+func DNSNames() ([]string, error) {
+	domainName, err := ioutil.ReadFile("/proc/sys/kernel/domainname")
+	if err != nil {
+		return nil, err
+	}
+
+	dnsNames := []string{}
+
+	if strings.Contains(string(domainName), "(none)") {
+		var hostname string
+
+		if hostname, err = os.Hostname(); err != nil {
+			return nil, err
+		}
+
+		dnsNames = append(dnsNames, hostname, fmt.Sprintf("%s.%s", hostname, domainName))
+	}
+
```

Comment:
don't we need to handle the case where domainName is not `none`?

Reply:
Eh I guess I misread the logic // comment.

Reply:
Not sure I follow. Although this could be a stale review since I just pushed changes. Shouldn't `if string(domainName) != "(none)\n"` cover what your saying?

Reply:
Want to change the conditional to be
```
if string(domainName) == "(none)\n" {
  return dnsNames, nil
}

var hostname ...
```

Reply:
Yep, much better.